src/main/java/com/clickhouse/kafka/connect/sink/db/helper/ClickHouseHelperClient.java

```diff
@@ -45,10 +45,10 @@ private ClickHouseNode create() {
 
         String url = String.format("%s://%s:%d/%s", protocol, hostname, port, database);
 
-        LOGGER.info("url: " + url);
+        LOGGER.info("ClickHouse URL: " + url);
 
         if (username != null && password != null) {
-            LOGGER.info(String.format("Adding username [%s] password [%s]  ", username, Mask.passwordMask(password)));
+            LOGGER.debug(String.format("Adding username [%s]", username));
```

Comment:
Why you removed the mask of the password from logging since in some cases it easy to troubleshot (i do not think we currently have a good error if the credentials are wrong)